packages/palette/src/elements/Cards/MediumCard.tsx

```diff
@@ -43,17 +43,15 @@ export const MediumCard: React.FC<MediumCardProps> = ({
           width="100%"
           height="100%"
         />
-        <Box position="absolute" bottom="15px" left="15px">
+        <Box position="absolute" bottom={1.5} left={1.5}>
           <Sans size="5t" weight="regular" color="white100">
             {title}
           </Sans>
           <Sans size="3t" weight="regular" color="white100">
             {subtitle}
           </Sans>
         </Box>
-        {!!tag && (
-          <CardTag {...tag} position="absolute" top="15px" left="15px" />
-        )}
+        {!!tag && <CardTag {...tag} position="absolute" top={1.5} left={1.5} />}
```

Comment:
Ah I just PR'd some changes here. I can deal with the conflicts.

Reply:
Sounds good. Relatively minor in either case.